cmd/internal/account/current_command.go

```diff
@@ -0,0 +1,35 @@
+package account
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/urfave/cli/v2"
+)
+
+func currentCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "current",
+		Usage: "Outputs the account you currently have selected",
+
+		// Use a space to cause the args usage to not be displayed since the `current` command
+		// doesn't accept any arguments
+		ArgsUsage: " ",
```

Comment:
This feels a bit hacky. I doubt it would cause any issues, but I'm happy to change it / do more investigation if folk aren't happy with it.